.github/styles/Vocab/Products/accept.txt

```diff
@@ -47,49 +84,112 @@ dev
 Dexalot
 Docusaurus
 dotenv
+drizzleOptions
 ecrecover
+Enums
+env
 EOAs
+ERC
+ERC721s
 ESLint
+Eteng
 Ethereum
+Ethers
+ethtx
```

Comment:
eth tx?

Reply:
Yes. It's a variable name which is called out in a markdown file.

Reply:
Resolved